lib/plugins.js

```diff
@@ -231,14 +231,20 @@ function plugins (settings, deviceId) {
   }
 
   function recordPing (serialNumber, deviceTime, rec) {
-    const r = {
+    const r_mp = {
```

Comment:
Should be camel-case. Also, if there are multiple variables, longer names are better.